packages/cli-core/API.md

```diff
@@ -6,10 +6,48 @@
 
 /// <reference types="node" />
 
+import { AmplifyIOHost } from '@aws-amplify/plugin-types';
+import { AmplifyIoHostEventMessage } from '@aws-amplify/plugin-types';
+import { AmplifyIoHostEventRequestMessageIoRequest } from '@aws-amplify/plugin-types';
 import { PackageManagerController } from '@aws-amplify/plugin-types';
 import { WriteStream } from 'node:tty';
 import z from 'zod';
 
+// @public
+export class AmplifyEventLogger {
+    constructor();
+    // (undocumented)
+    amplifyNotifications: <T>(msg: AmplifyIoHostEventMessage<T>) => Promise<void>;
+    // (undocumented)
+    cdkDeploymentProgress: <T>(msg: AmplifyIoHostEventMessage<T>) => Promise<void>;
+    debug: <T>(msg: AmplifyIoHostEventMessage<T>) => Promise<void>;
+    fancyCfnDeploymentProgress: <T>(msg: AmplifyIoHostEventMessage<T>) => Promise<void>;
+    // (undocumented)
+    getEventLoggers: () => {
+        notify: (<T>(msg: AmplifyIoHostEventMessage<T>) => Promise<void>)[];
+    };
+    nonTtyCfnDeploymentProgress: <T>(msg: AmplifyIoHostEventMessage<T>) => Promise<void>;
+}
+
+// @public
+export class AmplifyIOEventsBridge implements AmplifyIOHost {
+    constructor(eventHandlers: {
+        notify?: (<T>(msg: AmplifyIoHostEventMessage<T>) => Promise<void>)[];
+        requestResponse?: (<T, U>(msg: AmplifyIoHostEventRequestMessageIoRequest<T, U>) => Promise<U>)[];
+    });
+    notify<T>(msg: AmplifyIoHostEventMessage<T>): Promise<void>;
+    requestResponse<T, U>(msg: AmplifyIoHostEventRequestMessageIoRequest<T, U>): Promise<U>;
+}
+
+// @public (undocumented)
+export const amplifyIOEventsBridgeFactory: AmplifyIOEventsBridgeSingletonFactory;
+
+// @public
+export class AmplifyIOEventsBridgeSingletonFactory {
+    constructor();
+    getInstance: () => AmplifyIOEventsBridge;
+}

```

Comment:
Does `AmplifyIOEventsBridge` have to be public?
I.e. could factory return `AmplifyIOHost` ?

Reply:
Yup, I missed that. Updated